From 6cec272b572513e90135c3cf178f3eab2fa2dbbb Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Wed, 18 Jun 2014 15:42:23 +0200
Subject: [LIBREPORT PATCH 111/111] mailx: improve notification e-mail format

Make it more obvious who is the sender. Ensure that the notify*
configuration won't be overwritten by the configuration changes of the
mailx plugin. Distinguish between e-mails for a new crash and a new
occurrence.

The notify* configuration could be stored in a new configuration file
but such an approach seems a bit overengineered.

Resolves: rhbz#1093375

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/plugins/mailx_event.conf | 16 +++++++++++++---
 1 file changed, 13 insertions(+), 3 deletions(-)

diff --git a/src/plugins/mailx_event.conf b/src/plugins/mailx_event.conf
index f3176bd..88c53ec 100644
--- a/src/plugins/mailx_event.conf
+++ b/src/plugins/mailx_event.conf
@@ -1,6 +1,16 @@
-# This is already enabled in abrt_event.conf
-#EVENT=post-create
-    reporter-mailx --notify-only -c /etc/libreport/plugins/mailx.conf
+EVENT=notify
+    # do not rely on the default config nor on the config file
+    Mailx_Subject="[abrt] a crash has been detected" \
+    Mailx_EmailFrom="ABRT Daemon <DoNotReply>" \
+    Mailx_EmailTo="root@localhost" \
+    reporter-mailx --notify-only
+
+EVENT=notify-dup
+    # do not rely on the default config nor on the config file
+    Mailx_Subject="[abrt] a crash has been detected again" \
+    Mailx_EmailFrom="ABRT Daemon <DoNotReply>" \
+    Mailx_EmailTo="root@localhost" \
+    reporter-mailx --notify-only
 
 # Users should user rhtsupport plugin, so disabling others by default
 # to not confuse users with too many options
-- 
1.8.3.1

